src/domain/accounts/const.ts

```diff
@@ -0,0 +1,11 @@
+export const AccountLevel = {
```

Comment:
Up to now we are using a file called `primitives.ts` for these kind of types.

Reply:
updated